coretex/cli/utils.py

```diff
@@ -14,3 +34,56 @@ def arrowPrompt(choices: List[Any]) -> Any:
     ])
 
     return answers["option"]
+
+
+def checkIfExpired(expirationDate: datetime) -> bool:
+    currentTime = datetime.utcnow().replace(tzinfo=timezone.utc)
+    if currentTime >= expirationDate:
+        return True
+    return False
+
+
+def refresh() -> None:
+    config = loadConfig()
+    apiExpirationDate = decodeDate(config["tokenExpirationDate"])
```

Comment:
Name of the config property and name of the variable do not match, let's be consistent with naming.